src/python/pants/core/goals/test.py

```diff
@@ -316,6 +318,20 @@ def register_options(cls, register) -> None:
                 "system supports this."
             ),
         )
+        register(
+            "--extra-env-vars",
+            type=list,
+            member_type=str,
+            default=[],
+            help="Specify a list additional environment variables to include in test processes. Entries are strings "
+            "in the form `ENV_VAR=value` to use explicitly; or just `ENV_VAR` to copy the value of a variable in Pants' "
```

Comment:
Nitty nit: This made me consider whether "Pants", as we use it, is a singular or plural noun. The original noun "Pants" is plural ("his Pants were dry", not "his Pants was dry"), but in our usage of "Pants" as a proper noun, it is singular ("Pants is a cool build system", not "Pants are cool build systems"). So I think this should be "Pants's".

Reply:
Yeah I've been using `Pants's` in documentation.

Note that there is no consensus, though, on how the singular possessive works when the word ends in "s". https://www.dictionary.com/e/whats-the-rule-for-doing-a-possessive-after-the-word-s/ What a fun language 🙃 Either form is correct. Consistency is more important.